src/addTest.js

```diff
@@ -13,7 +13,8 @@ define(['ModernizrProto', 'Modernizr', 'hasOwnProp', 'setClasses'], function(Mod
    * @access public
    * @function on
    * @param {string} feature - String name of the feature detect
-   * @param {function} cb - Callback function returning a Boolean - true if feature is supported, false if not
+   * @param {Function} cb - Callback function returning a Boolean - true if feature is supported, false if not
+   * @returns {void}
```

Comment:
void !== undefined. This should be `@returns {undefined}`, right?

Reply:
According to https://eslint.org/docs/rules/valid-jsdoc: `// return type `void` means the function has no `return` statement`
Can change the voids to undefined though if you think thats clearer.